pkg/asset/rhcos/bootstrap_image.go

```diff
@@ -40,10 +40,12 @@ func (i *BootstrapImage) Generate(p asset.Parents) error {
 	config := ic.Config
 
 	// Check for RHCOS image URL override
-	if boi := config.Platform.BareMetal.BootstrapMachineOSImage; boi != "" {
-		logrus.Info("Found override for Bootstrap OS Image.")
-		*i = BootstrapImage(boi)
-		return nil
+	if config.Platform.Name() == baremetal.Name {
+		if boi := config.Platform.BareMetal.BootstrapMachineOSImage; boi != "" {
+			logrus.Info("Found an override in Baremetal Platform for Bootstrap Machine OS Image.")
```

Comment:
Same here, if we're going to log we should probably include the value we're overriding with?

Reply:
Sure thing. Coming up in the next commit.